whylog/tests/tests_log_reader/test_files/010_multiple_files/rules.yaml

```diff
@@ -1,4 +1,5 @@
 ---
 causes: [cause]
 effect: effect
-node: different
+constraints: []
+#node: different
```

Comment:
There is nothing common with Ewa stuff. In the past it meant that whylog should search many files. But now whylog by default search in many files.